images/cfw/opt/x1plus/lib/python/x1plus/utils.py

```diff
@@ -1,7 +1,24 @@
 import subprocess
 from functools import lru_cache
 import os
+import glob
+import re
+import sys
+import importlib
+import logging, logging.handlers
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+slh = logging.handlers.SysLogHandler(address="/dev/log")
+slh.setLevel(logging.INFO)
+slh.setFormatter(logging.Formatter("%(name)s: %(message)s"))
+logger.addHandler(slh)
+
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+ch.setFormatter(logging.Formatter("[%(asctime)s] %(name)s: %(levelname)s: %(message)s"))
+logger.addHandler(ch)
```

Comment:
or arguably this should just be hoisted into the x1plus package itself to make sure that we get the logger in everything x1plus-ish